modules/vector-search/pages/fine-tune-vector-search.adoc

```diff
@@ -0,0 +1,132 @@
+
+Couchbase dynamically adjusts two critical vector parameters:
+
+`nlist` (or `Centroid` count)::
+The number of clusters used for indexing.
+The centroids are used to quickly find the surrounding closest matches in the index. Increasing the number of centroids will increase accuracy but will decrease the speed of the search.
++
+The `nlist` is determined dynamically based on the size of the dataset (number of vectors) in a partition:
++
+[%header, cols="3*a"]
+|===
+| Number of vectors in partition (`nvec`)
+| `nlist` calculation
+| Notes
+
+| stem:["nvec " ge " 200,000"]
+| stem:[4 xx sqrt("nvec")]
+| This formula is designed to handle larger datasets
+ where increasing the number of datasets does not yield significant improvements in recall.
+
+| stem:["1000" lt "nvec" le "200,000" ]
+| stem:["nvec" / 100]
+| This formula targets approximately 100 vectors per cluster,
+which balances between too few and too many clusters, ensuring efficient indexing.
+
+| stem:["nvec" lt 1000]
+| N/A
+| For number of vectors less than 1000, Couchbase will carry out a straight forward one-to-one mapping between IDs and vectors with an exact vector comparison.
+Vectors are directly stored without the need for additional processing for the `nlist` calculation.
+
+|===
+
+`nprobes` (or `probes`)::
+The number of `centroids` that a query will check for similar vectors.
+The `nprobe` value is only set when Couchbase is using an Inverted File Index. Couchbase will select the best index type and comparison method depending on the size of the dataset and the `vector_index_optimized_for` setting.
++
++
+[%header, cols="3*a"]
+|===
+| Query optimization
+| `nprobe` calculation
+| Notes
+
+
+| Default calculation
+| stem:[sqrt("nlist")]
+| This provides a balanced tradeoff between recall and latency by adjusting the number of clusters probed during queries.
+
+| Latency-optimized (`vector_index_optimized_for: latency`)
+| stem:[sqrt("nlist") / 2]
+| A minimum value of 1 is enforced to avoid setting `nprobe` too low.
+
+|===
+
+
+== Fine-Tuning Query Parameters
+
+
+The Vector Search API will accept an additional parameter object, allowing for tuning the recall and/or accuracy of the search execution.
+
+`ivf_nprobe_pct`::
+Adjusts the number percentage of clusters searched during queries, allowing for fine-tuning of the balance between recall and performance.
+If the value of `nprobe` is 5% of `nlist` (the centroid count), then setting the value of `ivf_nprobe_pct` higher than 5% will have the search cover a higher percentage of clusters, which will improve the accuracy of the search.
+
+`ivf_max_codes_pct`::
+The default value for this is 100, and the value represents the percentage of `centroids` that will be visited during a search. Reducing the value reduces the number of centroids visited, which will descrease accurary and recal, but will result in faster compute times.
```

Comment:
```suggestion
You can add the following parameters to your query: 

`ivf_nprobe_pct`::
Adjusts the number percentage of clusters searched during queries, allowing for fine-tuning of the balance between recall and performance.
If the value of `nprobe` is 5% of `nlist` (the centroid count), then setting the value of `ivf_nprobe_pct` higher than 5% will have the search cover a higher percentage of clusters, which will improve the accuracy of the search.

`ivf_max_codes_pct`::
The default value for this is 100, and the value represents the percentage of `centroids` that will be visited during a search. 
Reducing the value reduces the number of centroids visited, which will decrease accuracy and recall, but will result in faster compute times.
```

These need to be added to the full Search documentation, with the explanation of all of the Search request parameters: https://docs.couchbase.com/cloud/search/search-request-params.html

If you don't feel up to doing that yourself, please log a ticket and I will pick it up.